Behavior/BehaviorDelegate.swift

```diff
@@ -25,10 +25,11 @@ class BehaviorDelegate: SpeziAppDelegate {
                 AccountConfiguration(configuration: [
                     .requires(\.userId),
                     .requires(\.name),
+                    .requires(\.dateOfBirth),
 
                     // additional values stored using the `FirestoreAccountStorage` within our Standard implementation
                     .collects(\.genderIdentity),
-                    .collects(\.dateOfBirth)
+                    .collects(\.email)
```

Comment:
The email key is still present and causes issues as outlined already in https://github.com/CS342/2024-Behavior/pull/5#discussion_r1457921826.

Reply:
Could you explain again where the email is popping up twice?